en/docs/tutorials/expose-an-existing-api-via-a-rest-api-proxy.md

```diff
@@ -16,13 +16,14 @@ Let's begin...
 
 Choreo gives you two options to create a REST API proxy: 
```

Comment:
The word "methods" seem more correct here than "options".

```suggestion
You can create a REST API proxy iin one of the following methods:
```

Reply:
done